src/python/pants/bin/pants_runner.py

```diff
@@ -32,12 +32,13 @@ def __init__(self, exiter, args=None, env=None, start_time=None):
     self._start_time = start_time
 
   def run(self):
+    ExceptionSink.reset_exiter(self._exiter)
     options_bootstrapper = OptionsBootstrapper(env=self._env, args=self._args)
     bootstrap_options = options_bootstrapper.get_bootstrap_options()
     global_bootstrap_options = bootstrap_options.for_global_scope()
+    ExceptionSink.reset_should_print_backtrace_to_terminal(
```

Comment:
nit: I think whitespace between the `ExceptionSink` commands and the options bootstrapping would make this a little easier to read.

Reply:
Done!